dockerfiles/init/manifests/che.env

```diff
@@ -170,7 +170,7 @@
 #    2. Else use the value of CHE_API
 #
 # Che Server --> Docker Daemon Progression:
-#    1. Use the value of CHE_DOCKER_DAEMON__URL
+#    1. Use the value of CHE_INFRA_DOCKER_
```

Comment:
Should `CHE_INFRA_DOCKER_DAEMON__URL` be here?

Reply:
yup :)